torch/quantization/fuse_modules.py

```diff
@@ -124,6 +124,10 @@ def fuse_known_modules(mod_list):
         raise NotImplementedError("Cannot fuse modules: {}".format(types))
     new_mod = [None] * len(mod_list)
     new_mod[0] = fuser_method(*mod_list)
+    # Move pre forward hooks of the base module to fused module
```

Comment:
what about post forward hook of the last module?

Reply:
Yes, I don't see any restriction to implement that.
When I proposed that I thought that base layers such as Linear and Conv2d have more in common than ending layers activation and batchnorm

Reply:
I think it's still better to include that just for completeness

Reply:
sure